### PR TITLE
Update count of Belgian municipalities

### PR DESCRIPTION
**Pull request by RVA2869**
# Description
The municipalities of [Bertogne ](https://www.wikidata.org/wiki/Q713118)and [Bastogne ](https://www.wikidata.org/wiki/Q34021)merged on December 2, 2024.
(There are many mergers taking place in January.)

### Type of Change

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation update
- [x] Code refactor

## Checklist

Please **ensure** the following before submitting the PR:

- [x] I have **tested** the changes locally, and they work as expected.
- [x] The code follows the project's coding standards and conventions.
- [x] Any **query changes** have been verified and are working correctly.

### CODE REVIEW

**Comment by RVA2869** [comment]
❗ PR currently on hold, a user decided to change the model...

**Comment by RVA2869** [comment]
@Ainali Thanks for the suggestion with the end date qualifier. But I went for  [Q713118#P576](https://www.wikidata.org/wiki/Q713118#P576).
Cleaning up redundant code ,and sort by typelabel instead of type orglabel. (better in this case)

File: queries/generators/belgium.rq [hunk 1]
```diff
@@ -56,4 +50,4 @@ UNION
 
   SERVICE wikibase:label { bd:serviceParam wikibase:language "en,nl,fr,de" }
 }
-ORDER BY ?type ?orgLabel
+ORDER BY ?typeLabel ?type
```
Comment (Ainali) [inline queries/generators/belgium.rq:53]:
```suggestion
ORDER BY ?typeLabel ?orgLabel
```
I buy that sorting on the label for type may make more sense, but then the type adds no value at all (as we will never have several types for each type label). Let's keep the alphabetical sorting of the orgs within each type.

**Review by Ainali — APPROVED** [review]
Great!

**Pull request merged**